### PR TITLE
Update release templates to used GenerateRelease notes automation

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Switch ios and cpp to use the generate release notes automation.

### CODE REVIEW

**Review by tjprescott — COMMENTED** [review]
LGTM. Thanks @chidozieononiwu!

File: releases/2021-06/ios.md [hunk 1]
```diff
@@ -5,131 +5,4 @@ tags: ios azure
...
-- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-ios/issues/new/choose).
-- Check [previous questions](https://stackoverflow.com/questions/tagged/azure+ios) or ask new ones on
- StackOverflow using the `azure` and `ios` tags.
-
-## Latest Releases
-
-View all the latest versions of iOS packages [here][ios-latest-releases].
-
-{% include refs.md %}
+{% include releases/notes/common.md language="ios" date="2021-06" displayDate="June 2021" %}
```
Comment (vcolin7) [inline releases/2021-06/ios.md:8]: We had made some manual changes to the iOS notes that will be lost with this change. I understand we can make the same changes to the CHANGELOG the contents here are pulled from but I was wondering how often these documents are generated. We could also simply not update the notes for June this time around :P

Comment (chidozieononiwu) [reply releases/2021-06/ios.md:8]: That's reasonable. I can undo the changes for this files.

**Review by weshaggard — APPROVED** [review]
Looks reasonable. As already stated since they were produced manually for this month already lets just make sure we get this merged before we start generating for next month. Lets also double check that the formatting looks correct next month when we start using this.

**Pull request merged**